packages/devextreme/testing/tests/DevExpress.ui.widgets.form/form.validationRules.tests.js

```diff
@@ -475,6 +475,33 @@ QUnit.test('Changing an validationRules options of an any item does not invalida
     assert.strictEqual(renderComponentSpy.callCount, 0, 'renderComponentSpy.callCount');
 });
 
+[false, true].forEach(hasValidationGroup => {
+    QUnit.test(`form ${hasValidationGroup ? 'with' : 'without'} validation group should validate without errors (T1233487)`, function(assert) {
```

Comment:
[test naming]
Scenario is not completely described. 
```suggestion
    QUnit.test(`form ${hasValidationGroup ? 'with' : 'without'} validationGroup property specified should validate without errors after nested validators remove (T1233487)`, function(assert) {
```

Reply:
[daf73a4](https://github.com/DevExpress/DevExtreme/pull/27435/commits/daf73a4cac0e007e5fc9a5f13c3cb1498f71d603)